### PR TITLE
Add Revision Support Feature Documentation

### PR DESCRIPTION
**Pull request by tharikaGitHub**
## Purpose
This PR adds documentation for the Revision Support Feature. Please find the document screenshots below.

### CODE REVIEW

**Comment by tmkasun** [comment]
I think some of these screenshots will change after addressing the code review suggestion we got during the last review session, For example, Show an icon inside the review circles indicating whether it is deployed somewhere (to an environment) or not

**Comment by tharikaGitHub** [comment]
> I think some of these screenshots will change after addressing the code review suggestion we got during the last review session, For example, Show an icon inside the review circles indicating whether it is deployed somewhere (to an environment) or not

Yes @tmkasun this is to add documentation for the milestone release so the users can try it out with the existing UI. We will change the screenshots after addressing the review concerns.

**Review by CrowleyRajapakse — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]

File: en/docs/design/create-api/create-api-revisions.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## Delete a revision
+
+If you need to delete a specific revision, click on the **Delete** button in the revision list as shown below. If a revision is already deployed in a specific environment, you cannot delete it. You should first undeploy the revision from the Gateway Environment and then delete it.
+
+[![Delete a revision]({{base_path}}/assets/img/design/revision/delete-revision.png)]({{base_path}}/assets/img/design/revision/delete-revision.png)
+
+## Restore a revision
+
+If you need to restore a specific revision, click on the **Restore** button in the revision list as shown below. This will restore the revision in to the working copy.
```
Comment (Mariangela) [inline en/docs/design/create-api/create-api-revisions.md:87]: Is the terminology "working copy" correct here? Or has a new name been decided on?

**Review by Mariangela — APPROVED** [review]

File: en/docs/design/create-api/create-api-revisions.md (hunk 1 above)
Comment (tharikaGitHub) [reply en/docs/design/create-api/create-api-revisions.md:87]: Yes this should be changed.

Comment (tharikaGitHub) [reply en/docs/design/create-api/create-api-revisions.md:87]: Shall we change it to ```This will restore the API with the specific revision.```?

Comment (Mariangela) [reply en/docs/design/create-api/create-api-revisions.md:87]: @tharikaGitHub  +1 to the suggested change

**Pull request merged**